scripts/cipublish

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+
+Publish container images built from templates.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        docker login -u "${QUAY_USER}" -p "${QUAY_PASSWORD}" quay.io
+
+        docker push "quay.io/azavea/openjdk-gdal:$GDAL_VERSION-$VARIANT"
```

Comment:
I created a stub repository in Quay for this at https://quay.io/repository/azavea/openjdk-gdal. We'll still need to get the credentials for the `azavea+images` robot and feed them to Travis CI so that the login command above can succeed.

Reply:
I temporarily set Travis to deploy from this branch in https://github.com/azavea/docker-openjdk-gdal/pull/1/commits/85a30ec017038408e2d4ec89a57d272a95b93006 to make sure that this would work. Images are up here: https://quay.io/repository/azavea/openjdk-gdal The commit has been reverted, so we should only deploy from `master` now.